okhttp/src/main/java/com/squareup/okhttp/CertificatePinner.java

```diff
@@ -123,17 +123,45 @@ private CertificatePinner(Builder builder) {
    * pinned for {@code hostname}. OkHttp calls this after a successful TLS
    * handshake, but before the connection is used.
    *
+   * @return {@code true} if hostname has pinned certificates and check passed,
```

Comment:
Just to be sure: you mean remove it from javadoc or remove it from the code too? (without this boolean tests won't be able to check that behavior is correct, but it's okay for `CertificatePinner#check()` general contract), I agree that it's a bit confusing